### DISCUSSION TITLE
Will there be a chance to replace libbass?

### DISCUSSION THREAD

**Discussion by Catty2014** [ppy/osu#29939] [Q&A] [answered]
Replacing an audio engine might be a huge effort and may cause unstable gameplay, but if we could replace audio engine with an open-source one, porting would be much easier.
So, will there be a chance to replace libbass with open-source ones like SoLoud?

**Comment by Joehuu** [comment, marked as answer]
Relevant framework issue: https://github.com/ppy/osu-framework/issues/2784
